### PR TITLE
Built-in Sandbox: add more unit tests for dragonball. Part 4

### PR DESCRIPTION
**Pull request by wllenyj**
Added more unit tests for Vcpu Manager.

Fixes: #4899 

Signed-off-by: wllenyj <wllenyj@linux.alibaba.com>

### CODE REVIEW

**Review by bergwolf — APPROVED** [review]

File: src/dragonball/src/vcpu/vcpu_impl.rs [hunk 1]
```diff
@@ -813,11 +811,8 @@ pub mod tests {
     #[cfg(target_arch = "x86_64")]
     fn create_vcpu() -> (Vcpu, Receiver<VcpuStateEvent>) {
         // Call for kvm too frequently would cause error in some host kernel.
-        std::thread::sleep(std::time::Duration::from_millis(5));
```
Comment (liubin) [inline src/dragonball/src/vcpu/vcpu_impl.rs:816]: Should line 815 be deleted too?

**Review by liubin — APPROVED** [review]

File: src/dragonball/src/vcpu/vcpu_impl.rs (hunk 1 above)
Comment (wllenyj) [reply src/dragonball/src/vcpu/vcpu_impl.rs:816]: removed

**Pull request merged**